File: JAVA PROGRAMMING/Test74.java
// 74. CONVERT DECIMAL NUMBER TO ANY BASE OF NUMBER
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Decimal Value: ");
        int num = in.nextInt();

        System.out.println("Decimal To Binary Number: "+Integer.toBinaryString(num));
        System.out.println("Decimal To Octal Number: "+Integer.toOctalString(num));
        System.out.println("Decima To Hexadecimal Number: "+Integer.toHexString(num));
        
        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test74.js
"use strict";
// 74. CONVERT DECIMAL NUMBER TO ANY BASE OF NUMBER
const Convert = (num) => {
    console.log(`Decimal Number Is: ${num}`);
    console.log(`Decimal To Binary: ${num.toString(2)}`);
    console.log(`Decimal To Octal: ${num.toString(8)}`);
    console.log(`Decimal To Hexa Decimal: ${num.toString(16)}`);
    console.log();
};
Convert(255);
Convert(3526);
Convert(66);